blog(release): optimize 3.0 alpha2 draft (#119)
camunda/blog.camunda.org: content/post/2020/02/camunda-optimize-30-alpha2-released.md

* Support for Elasticsearch 7.6

The [complete release notes]( https://jira.camunda.com/secure/ReleaseNote.jspa?projectId=10730&version=15891) are available in Jira.

<!--more-->

You can [try out a free trial of Camunda Optimize](#how-to-get-it).

# Enhanced User Task Monitoring and Reporting

Optimize 3.0-alpha2 includes new User Task Monitoring and Reporting capabilities.

Up to this point when monitoring the distribution of UserTasks across Assignees or Candidate Groups, it was only possible to see the number or duration of each User Task per Assignee or Candidate Group:

{{< figure src="usertask-report1.png" alt="User Task Group by Assignee distribute by Flow Node " >}}

With this release we add the possibility to group by Flow Node and distributed by Assignee and Candidate Group. The resulting report could look like this:
{{< figure src="usertask-report2.png" alt="User Task Group by Flow Node distribute by Assignee" >}}

This report can help to get a good overview about the distribution of each User Task across different Assignees or Candidate Groups and makes Optimize's User Task Reporting capabilities even stronger.

# Improved Dashboard Handling

> In addition to the new report capabilities, we also improved the way you can work with dashboards. When adding a report to a dashboard other reports automatically make space for the report. Especially in dashboards that contain many reports this improvement simplifies dashboard interactions a lot.  

Reviewer: ```suggestion
In addition to the new reporting capabilities, this release also improves the way you can work with dashboards. When adding a report to a dashboard other reports automatically make space for the report. Especially in dashboards that contain many reports this improvement simplifies dashboard interactions a lot.  
```

{{< video mp4="dashboard-behaviour.mp4" alt="Improved Dashboard Behaviour" >}}

# Support for Elasticsearch 7.6

> From this version we are now officially supporting Elasticsearch 7.0.x - 7.6.x.

Reviewer: ```suggestion
From this version on, Elasticsearch 7.0.x - 7.6.x is now officially supported.
```

# What's Next?

> We are getting closer to the next major release of Camunda Optimize which will be released beginning of April. Stay tuned.

Reviewer: ```suggestion
The next major release of Camunda Optimize is getting closer and will be released beginning of April. Stay tuned!
```

> A smooth update from Camunda Optimize 2.7 to Optimize 3.0 will be possible. Along with the release we will publish a dedicated update guide.

Reviewer: ```suggestion
A smooth upgrade from Camunda Optimize 2.7 to Optimize 3.0 will be possible. Together with the final 3.0.0 release we will publish a dedicated upgrade guide.
```
